mmv1/products/networkservices/MulticastDomainActivation.yaml

```diff
@@ -93,13 +105,32 @@ properties:
       Use the following format:
       `projects/*/locations/global/multicastDomains/*`.
     immutable: true
+    required: true
```

Comment:
Could you just verify that this is safe? I'm guessing the resource could not be used without setting this field.